include/af/defines.h

```diff
@@ -357,6 +357,15 @@ typedef enum {
 } af_image_format;
 #endif
 
+#if AF_API_VERSION >=34
+typedef enum {
+    M00 = 0,
```

Comment:
Another point here is do you want to use moment or moments? Most places I've seen, you are using moments.